x-pack/platform/plugins/shared/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/analytics_detail_flyout/analytics_detail_flyout.tsx

```diff
@@ -103,8 +103,8 @@ export const AnalyticsDetailFlyout = () => {
           <EuiFlexItem grow={false}>
             <EuiButtonEmpty onClick={redirectToAnalyticsList}>
               <FormattedMessage
-                id="xpack.ml.jobDetailsFlyout.openJobsListButton"
-                defaultMessage="Open jobs list"
+                id="xpack.ml.overview.analyticsList.manageJobsButtonText"
```

Comment:
Is there an issue with this component in the DFA Analytics Map view? I'm clicking on the link here but not flyout opens. Can't see any errors in the browser console.

<img width="1047" height="468" alt="Screenshot 2025-09-19 at 13 09 28" src="https://github.com/user-attachments/assets/d29f7f88-40c7-4323-b3a9-876d2849f4a6" />